BetterHolidays/markets/nasdaq.py

```
@@ -0,0 +1,160 @@
+import datetime as dt
+from ..days import Day, Holiday, TradingDay, PartialTradingDay, NonTradingDay
+from ..const import MONTHS_MAP
+from ..utils import iter_year
+from .market import Market
+from .holidays import (
+    NewYearsDay,
+    MartinLutherKingJrDay,
+    WashingtonsBirthday,
+    LincolnsBirthday,
```

Ruff (0.8.2)

`.holidays.LincolnsBirthday` imported but unused

Remove unused import: `.holidays.LincolnsBirthday`

(F401)